pkg/webhook/mutating_webhook.go

```diff
@@ -51,3 +58,50 @@ func (m *MutateFunc) Mutate(ctx context.Context, req admission.Request) admissio
 
 	return m.MutatingWebhook.Mutate(ctx, req)
 }
+
```

Comment:
Can we leave the handler clean with only two interfaces (i.e. `Mutator` and `Validator`)?

I would rather suggest to embed and extend the `MutatingWebhook`:
```
type ObjectMutatingWebhook struct {
	MutatingWebhook
}


func (o *ObjectMutatingWebhook) Mutate(ctx context.Context, req admission.Request) admission.Response {
	obj := req.Object.Object
	err := o.MutateObject(ctx, req, obj)
	if err != nil {
		return admission.Errored(http.StatusInternalServerError, err)
	}
	marshalled, err := json.Marshal(obj)
	if err != nil {
		return admission.Errored(http.StatusInternalServerError, err)
	}
	return admission.PatchResponseFromRaw(req.Object.Raw, marshalled)
}

func (o *ObjectMutatingWebhook) MutateObject(_ context.Context, _ admission.Request, _ runtime.Object) error {
	return nil
}
```